drivers/include/enc28j60.h

```diff
@@ -39,21 +39,17 @@ typedef struct {
     spi_t spi;              /**< If I drink */
     gpio_t cs_pin;          /**< beer in the evening, */
     gpio_t int_pin;         /**< I will be most certainly */
-    gpio_t reset_pin;       /**< drunk in the morning?! */
+    gpio_t rst_pin;         /**< drunk in the morning?! */
 } enc28j60_params_t;
 
 /**
  * @brief   ENC28J60 device descriptor
  */
 typedef struct {
-    netdev_t netdev;        /**< pull in the netdev fields */
-    spi_t spi;              /**< SPI bus the transceiver is connected to */
-    gpio_t cs_pin;          /**< pin connected to the CHIP SELECT line */
-    gpio_t int_pin;         /**< pin connected to the INT line */
-    gpio_t reset_pin;       /**< pin connected to the RESET line */
-    mutex_t devlock;        /**< lock the device on access */
-    int8_t bank;            /**< remember the active register bank */
-    uint32_t tx_time;       /**< last transmission time for timeout handling */
+    netdev_t netdev;            /**< pull in the netdev fields */
+    enc28j60_params_t p;        /**< SPI and pin confiuration */
```

Comment:
could this be made a const pointer?

Reply:
mhm good point, but currently its done this way in all drivers. Nevertheless, it should safe bytes and time to simply use a (const) pointer here. But when we don't copy here anymore, is this safe memory-wise? I mean if we use the params headers with arrays it sure is but one could pass a params struct and though it away afterwards, or not?

Reply:
true, nevermind.

Reply:
mhm, it also increases ROM/RAM usage by some 30 Bytes, which I don't understand as I would've guessed that using a pointer instead of copying the struct should safe some bits and bytes